Deleted broken link and relocated content (#546)
knative/community: docs/README.md

- [How to contribute](https://github.com/knative/docs/tree/main/CONTRIBUTING.md) -- Contains information about how
  to contribute and outlines the roles for Docs contributors.

- Use one of the new content templates:

>    - [Documenation](https://github.com/knative/docs/tree/main/template-docs-page.md) -- Instructions and a template that

Reviewer: typo: documentation

>    - [Documenation](https://github.com/knative/docs/tree/main/template-docs-page.md) -- Instructions and a template that

Reviewer 2: ```suggestion
   - [Documentation](https://github.com/knative/docs/tree/main/template-docs-page.md) -- Instructions and a template that
```

>    - [Documenation](https://github.com/knative/docs/tree/main/template-docs-page.md) -- Instructions and a template that

Reviewer 2: ```suggestion
   - [Documentation](https://github.com/knative/docs/tree/main/template-docs-page.md) -- Instructions and a template that
```

>      you can use to help you add new documentation.

Reviewer 2: ```suggestion
     you can use for adding new documentation.
```

   - [Blog](https://github.com/knative/docs/tree/main/template-blog-entry.md) -- Instructions and a template that

>      you can use to help you post to the Knative blog.

Reviewer 2: ```suggestion
     you can use for posting to the Knative blog.
```
